packages/sui-bundler/webpack.config.dev.js

```diff
@@ -102,4 +107,4 @@ const webpackConfig = {
     config.sourcemaps && config.sourcemaps.dev ? config.sourcemaps.dev : 'none'
 }
 
-module.exports = webpackConfig
+module.exports = smp.wrap(webpackConfig)
```

Comment:
`SpeedMeasurePlugin` has `disable` option but I think that it could be better not to use it if `config.measure` is not true:

```
module.exports = config.measure ? smp.wrap(webpackConfig) : webpackConfig
```